js/XAPI.js

```diff
@@ -452,6 +461,11 @@ class XAPI extends Backbone.Model {
     });
   }
 
+  removeEventListeners() {
+    // Stop listening to all events
+    this.stopListening();
+  }
```

Comment:
Is there a need for this method if it only calls an existing function anyway?
Maybe line 257 should just be `this.stopListening();`

Reply:
I thought it safer to encapsulate this and allow it to be reusable and self-documenting